src/fillTank.js

```diff
@@ -26,10 +26,11 @@ function fillTank(customer, fuelPrice, amount = Infinity) {
 
   customer.vehicle.fuelRemains += roundedAmount;
   customer.money -= roundPrice(roundedAmount * fuelPrice);
+  customer.money = +customer.money.toFixed(2);
```

Comment:
The line `customer.money = +customer.money.toFixed(2);` is correctly rounding the customer's money to two decimal places, ensuring that the money is represented accurately as per the task requirements.